pkgs/development/python-modules/algopy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, lib
+, fetchurl
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "algopy";
+  version = "0.5.7";
+
+  src = fetchurl {
+    url = https://files.pythonhosted.org/packages/1e/44/e22102b72f0eb3118d4a9e4815104515e1996c3695c82047eeede8a41f89/algopy-0.5.7.zip;
+    sha256 = "1pxphhajk6b6kd0ich6jfwrvjv3rw7vz7dswb2iqz1g3zivgcmb9";
+  };
+  propagatedBuildInputs = [ numpy scipy ];
+
+  doCheck = false;
```

Comment:
Try to enable tests, or use [pythonImportsCheck](https://nixos.org/manual/nixpkgs/stable/#using-pythonimportscheck) at least because there's no compilation involved, python packages tend to be built fine even if they won't work later.

Reply:
Oof, when trying to use the provided unit tests, I discovered that they fail, both for numdifftools and algopy. I implemented the `pythonImportsCheck` variant.

Reply:
Do you know why they fail? Unit tests failing due to the nix sandbox is usually fine to ignore, but it's always good to look into it. Maybe I can help if you post the failures here.

Reply:
The tests for algopy fail due to a change in the numpy api. I opened an issue over at the algopy repo, but got no response so far.

For numdifftools the last 5 tests fail, but it appears to have nothing to do with nix. I will open an issue over at their repo as well. I tried to attach the log, but got an error from GitHub. If you want me to, I can still share it as a comment, but I don't see the need to at the moment.

Reply:
Alright, make sure to link the created issues near the `doCheck = false` in a comment, so people touching the nix expression in the future will be informed.

Reply:
So I could resolve the issue with numdifftools, I missed a dependency. Now the unittests for numdifftools pass. Sorry for not taking care of this before opening the pull request...